developers-landing.md

```diff
@@ -0,0 +1,12 @@
+---
+Id: developers-landing

```

Comment:
```suggestion
id: developers-landing

```

Reply:
@DaveDuggins as we talked, you need to update the related link for redirection to this new page in the `about.md` file. Ping me if you need any help.

Reply:
I'd prefer to take care of that after the page is posted. That way, when the link is live, the page is live, too.

Reply:
> That way, when the link is live, the page is live, too.

If you update the related link in this patch simultaneously, this purpose can be achieved.

Reply:
BTW, this file lives outside the site folder. I think it's unintended?

Reply:
Yes, it's unintended.